js/core/pci/scan.js

```diff
@@ -340,6 +340,7 @@ var pciSpace = (function(pciAccessorFactoryArg) {
  */
 var codeNameResolver = (function() {
   var classCodes = [
+    'RNG Device',
```

Comment:
this doesn't look right (Class codes section in http://wiki.osdev.org/PCI). RNG device is probably should be Unclassified here